app/pb_kits/playbook/pb_nav/_horizontal_nav.scss

```diff
@@ -0,0 +1,67 @@
+
+$selector: ".pb_nav_list";
+[class^=pb_nav_list][class*=_horizontal] {
+  ul {
+    display: flex;
+    align-items: center;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+  }
+
+  [class*=_border_item] {
+    transition-property: border-color;
+    transition-duration: 0.15s;
+    transition-timing-function: cubic-bezier(0.7, 0, 0.3, 1);
```

Comment:
@jasoncypret @markiearnold what are your thoughts on variable-izing any of the animation properties like this one regarding animation timing?

Reply:
@thestephenmarshall Good catch - we do have [animation-curves tokens](https://github.com/powerhome/playbook/blob/master/app/pb_kits/playbook/tokens/_animation-curves.scss) for this and this should probably be using those, or maybe those should be adjusted @kre8sions

Reply:
Yes lets get this updated